client/snippets/aqua.json

```diff
@@ -1,6 +1,6 @@
 {
-    "import": {
-        "prefix": "import",
+    "aqua_imports": {
+        "prefix": "aqua_imports",
```

Comment:
Since, `terra` package will have the basic imports it is better if we have `aqua imports` distinguished. Although, the naming can be discussed.